app/models/push.rb

```diff
@@ -1,6 +1,9 @@
+# frozen_string_literal: true
+
 class Push < ActiveRecord::Base
   fields do
     status :string, limit: 32
+    email_sent :boolean, default: false
```

Comment:
probably not worth changing now, but slightly better would be to use a timestamp here (and allow nil, to indicate not sent). Then the UI could say for example: `Email was already sent at 3:45pm Mar 12, 2020`